Server/server.js

```diff
@@ -72,6 +73,32 @@ app.get('/products/:product/', function (req, res) {
   })  
 })
 
+app.post('/products/add/product', function (req, res) {
```

Comment:
Totally not necessary, but I noticed that you are querying 3 times for this post request.  If you're interested in increasing the number automatically in postgres without having to query 3 times for every post, check out these docs.  Let me know if you have any questions and I can help with that. https://www.postgresqltutorial.com/postgresql-sequences/